File: code-coaches/medium/building-blocks/building_blocks.cpp
#include <iostream>

int main()
{
    const int num_students = 15;
    int num_blocks = 0;
    for (int i = 0; i < 4; ++i)
    {
        int input;
        std::cin >> input;
        num_blocks += input;
    }

    std::cout << num_blocks % num_students << std::endl;

    return 0;
}

File: code-coaches/medium/building-blocks/BuildingBlocks.java
import java.util.Scanner;

public class BuildingBlocks {

    public static void main(String[] args) {
        final int numStudents = 15;
        int numBlocks = 0;
        Scanner scanner = new Scanner(System.in);
        for (int i = 0; i < 4; ++i) {
            int input = scanner.nextInt();
            numBlocks += input;
        }

        System.out.println(numBlocks % numStudents);

        scanner.close();
    }

}
